.env

```diff
@@ -0,0 +1,9 @@
+FLASK_APP=server.py
+
+FLASK_ENV=development
+
+POSTGRES_USER=user
+POSTGRES_PW=password
+POSTGRES_HOST=127.0.0.1
+POSTGRES_PORT=5432
+POSTGRES_DB=db_name
```

Comment:
Better to rename this file to something like `.env.example` and add an explanation to the `README.md` that developer needs to create `.env` manually. The file `.env` shouldn't be in the repository.